Makefile

```diff
@@ -187,19 +188,18 @@ help-verbose:
 # Virtual Environment
 ###############################################################################
 
-VENV = venv
-PYTHON_VENV = $(VENV)/bin/python3
-PIP_VENV = $(VENV)/bin/pip3
-CODESPELL_VENV = $(VENV)/bin/codespell
-MDFORMAT_VENV = $(VENV)/bin/mdformat
-MKDOCS_VENV = $(VENV)/bin/mkdocs
+VENV = .venv
+UV_RUN = uv run
 
-# Make a virtual environment.
-$(VENV):
-	@python3 scripts/check_python_version.py
-	@echo "Creating virtual environment"
-	@python3 -m venv $(VENV)
-	@$(PIP_VENV) install --quiet --upgrade uv
+# Sync dependencies using uv.
+sync: MAYBE_VERBOSE := $(if $(filter true,$(verbose)),--verbose)
+sync: pyproject.toml
```

Comment:
We need to add this rule (`sync`) to the help output. Or make it private (`_sync`).

Reply:
Good point. No need for a user "shortcut" imo; made it private.